backend/src/backend/app.py

```diff
@@ -85,6 +115,20 @@ async def get_tasks(user_id: Annotated[str, Depends(get_user_id)]):
     return [{"id": 1, "title": "Buy Milk"}, {"id": 2, "title": "Buy Bread"}]
 
 
+# just to demonstrate the db connection
+# TODO: Remove this endpoint once we have the actual endpoints
+# ref: https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg.connection.Connection
+@app.get("/test-db")
```

Comment:
Because this endpoint is expected to be deleted in the near future, I didn't use this to test the database connection. I will elaborate the test `test_db_connection()` once I come up with better approaches